### ISSUE TITLE
Command failed: npx prettier

### ISSUE THREAD

**Issue by robixdf** [pankod/superplate#266]
Hi all,

I'm getting this error when building a nextjs project with superplace-cli.

It looks like the process is trying to apply Prettier using project folder.
I'm running it on a MacOS 10.15.7.
Prettier: vers. 2.3.2

Command: npx superplate-cli superplate-next
✔ Cloned remote source successfully.
✔ Select your project type › nextjs
✔ What will be the name of your app · superplate-next
✔ Package manager: · yarn
✔ UI framework: · chakra-ui
✔ CSS Preprocessor: · css
✔ Features: · storybook,env
✔ Hooks · react-use
✔ State Management: · redux
✔ Do you want to use RTK Query ? · rtk-query
✔ i18n - Internationalization · next-i18next
✔ Linting tools: · eslint,prettier
✔ Do you want to use lint-staged? · none
✔ Testing Framework: · none
✔ E2E Testing framework: · none
✔ Docker integration: · none
✔ Continuous integration: · none
superplate-cli has encountered an error.

```
ERROR Error: Command failed: npx prettier /Users/robix/Downloads/dev/superplate-next --write
[error] No matching files. Patterns tried: /Users/robix/Downloads/dev/superplate-next !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**

    at ChildProcess.exithandler (child_process.js:319:12)
    at ChildProcess.emit (events.js:376:20)
    at maybeClose (internal/child_process.js:1055:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5) {
  killed: false,
  code: 2,
  signal: null,
  cmd: 'npx prettier /Users/robix/Downloads/dev/superplate-next --write',
  stdout: '',
  stderr: '[error] No matching files. Patterns tried: /Users/robix/Downloads/dev/superplate-next !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**\n'
}
```

Thank you for your help!

**Comment by omeraplak** [comment]
Hi @robixdf ,
Thank you for contacting us. We've fixed your issue and release 1.2.2. Could you try again?

**Comment by robixdf** [comment]
I've checked it out, it works fine now. Thanks!
